src/SFML/Graphics/Font.cpp

```diff
@@ -91,31 +92,28 @@ std::uint64_t combine(float outlineThickness, bool bold, std::uint32_t index)
 namespace sf
 {
 ////////////////////////////////////////////////////////////
-class Font::FontHandles
+struct Font::FontHandles
 {
-private:
-    // Default constructible deleter functor
-    struct Deleter
+    FontHandles() = default;
+
+    ~FontHandles()
     {
-        void operator()(FT_Library theLibrary)
-        {
-            FT_Done_FreeType(theLibrary);
-        }
-        void operator()(FT_Face theFace)
-        {
-            FT_Done_Face(theFace);
-        }
-        void operator()(FT_Stroker theStroker)
-        {
-            FT_Stroker_Done(theStroker);
-        }
-    };
+        FT_Stroker_Done(stroker);
+        FT_Done_Face(face);
+        // `streamRec` doesn't need to be explicitly freed.
+        FT_Done_FreeType(library);
+    }
+
+    FontHandles(const FontHandles&)            = delete;
+    FontHandles& operator=(const FontHandles&) = delete;
 
-public:
-    std::unique_ptr<std::remove_pointer_t<FT_Library>, Deleter> library;   //< Pointer to the internal library interface
-    std::unique_ptr<FT_StreamRec>                               streamRec; //< Pointer to the stream rec instance
-    std::unique_ptr<std::remove_pointer_t<FT_Face>, Deleter>    face;      //< Pointer to the internal font face
-    std::unique_ptr<std::remove_pointer_t<FT_Stroker>, Deleter> stroker;   //< Pointer to the stroker
+    FontHandles(FontHandles&&)            = delete;
+    FontHandles& operator=(FontHandles&&) = delete;
```

Comment:
These guarantee that the destructor is good enough for cleanup, as we don't ever perform ownership transfer.